### ISSUE TITLE
Caching

### ISSUE THREAD

**Issue by dozel** [alinz/react-native-webview-bridge#205]
Does this library use the same cache for each WebViewBridge? What I'm asking is, each time I open a new WebViewBridge, does it use its own cache, or do all of them share cache?

**Comment by dozel** [comment]
Could this be related -> https://github.com/facebook/react-native/pull/8578

**Comment by adamkdean** [comment]
I'm not sure, but there seriously needs to be a way to control the cache of WebView. Each of these PRs just keep getting closed without any action.

**Comment by aqnaruto** [comment]
yes, we are  seriously needs to be a way to control the cache of WebView, thank you

**Comment by vvavepacket** [comment]
any workaround for this? i need my webview to refresh the webpage it has right now (invalidate the cache),,, its not reflecting latest html/js code....
